### PR TITLE
Add torque test odrive configs

### PR DESCRIPTION
**Pull request by matthew-reynolds**

### CODE REVIEW

File: odrive/torque_test/m12.json [hunk 1]
```diff
@@ -0,0 +1 @@
..._undervoltage_trip_level": 8.0, "dc_max_negative_current": -0.009999999776482582, "dc_max_positive_current": Infinity, "enable_brake_resistor": true, "enable_can_a": true, "enable_dc_bus_overvoltage_ramp": false, "enable_i2c_a": false, "enable_uart_a": true, "enable_uart_b": false, "enable_uart_c": false, "error_gpio_pin": 0, "gpio10_mode": 11, "gpio11_mode": 2, "gpio12_mode": 12, "gpio13_mode": 12, "gpio14_mode": 2, "gpio15_mode": 7, "gpio16_mode": 7, "gpio1_mode": 4, "gpio2_mode": 4, "gpio3_mode": 3, "gpio4_mode": 3, "gpio5_mode": 3, "gpio6_mode": 0, "gpio7_mode": 0, "gpio8_mode": 0, "gpio9_mode": 11, "max_regen_current": 0.0, "uart0_protocol": 3, "uart1_protocol": 3, "uart2_protocol": 3, "uart_a_baudrate": 115200, "uart_b_baudrate": 115200, "uart_c_baudrate": 115200, "usb_cdc_protocol": 3, "gpio1_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio2_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio3_analog_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio3_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio4_analog_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio4_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}}}
```
Comment (wmmc88) [inline odrive/torque_test/m12.json]: Pls autoformat so that it's readable and not a single line.

**Review by wmmc88 — CHANGES_REQUESTED** [review]

File: odrive/torque_test/m12.json (hunk 1 above)
Comment (matthew-reynolds) [reply odrive/torque_test/m12.json]: no u

Comment (matthew-reynolds) [reply odrive/torque_test/m12.json]: Done in f52cba8

File: odrive/torque_test/m12.json [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+        "mirror_ratio": 1.0,
+        "pos_gain": 20.0,
+        "spinout_electrical_power_threshold": 10.0,
+        "spinout_mechanical_power_threshold": -10.0,
+        "steps_per_circular_range": 1024,
+        "torque_mirror_ratio": 0.0,
+        "torque_ramp_rate": 0.009999999776482582,
+        "vel_gain": 0.1666666716337204,
+        "vel_integrator_gain": 0.3333333432674408,
+        "vel_integrator_limit": Infinity,
```
Comment (wmmc88) [inline odrive/torque_test/m12.json:84]: Even github knows this is an invalid value lel

**Review by wmmc88 — APPROVED** [review]

**Pull request merged**